File: 프로그래머스/2/12985. 예상 대진표/예상 대진표.js
function solution(n,a,b)
{
    let count = 1;
    while(true){
        if(Math.abs(a-b)===1 && Math.max(a,b)%2===0){
            return count;
        }
        count += 1
        if(a%2===0) a /= 2
        else a = (a+1)/2
        
        if(b%2===0) b /= 2
        else b = (b+1)/2
    }

}

File: 프로그래머스/2/12985. 예상 대진표/예상 대진표.java
class Solution {
    public int solution(int n, int a, int b){
        int A = a;
        int B = b;
        int answer = 0;
        while(true) {
            if (A==B) {
                break;
            }

            A = (int)Math.ceil((double) A/2);
            B = (int)Math.ceil((double) B/2);
            answer++;
        }
        return answer;
    }
}